setup/github-bot/ansible-playbook.yaml

```diff
@@ -0,0 +1,73 @@
+---
+- hosts: github-bot
+  remote_user: root
+  gather_facts: False
+
+  tasks:
+
+    - name: Bootstrap | Install baseline packages
+      raw: apt-get install -y aptitude python-apt apt-transport-https
```

Comment:
@jbergstroem something like this, or would you rather see some fancy `when`-checks like what you put in the [fedora setup](https://github.com/nodejs/build/pull/446/files#diff-54a20c9aff06229996d3d614d66b5425R15)?